test/classes/nilability/nilable-equalities.chpl

```diff
@@ -2,11 +2,11 @@
 class MyClass { var x:int; }
 
 type od = owned MyClass;
-type ob = owned MyClass!; // aka borrowed MyClass
+type ob = owned MyClass!; // '!' binds more tightly than 'owned' et al.
```

Comment:
Please change this to `type ob = (owned MyClass)!` and undo the changes elsewhere in the file